### PR TITLE
feat(subscription/max_api_calls): update max_api_calls from chargebee subscription

### PR DESCRIPTION
**Pull request by gagantrivedi**

### CODE REVIEW

**Comment by gagantrivedi** [comment]
Open Question: are we okay with the default value of max_api_calls? Or should we update it with a data migration to appropriate values based on the `plan-id`? 
Although I don't think it will be a simple select/update since we have custom api_call for some plans as well?

**Comment by gagantrivedi** [comment]
Implements backend part of #631

File: api/organisations/tests/test_chargebee.py [hunk 1]
```diff
@@ -7,21 +7,24 @@
...
     get_subscription_data_from_hosted_page,
 )
 
 
 class MockChargeBeePlanResponse:
-    def __init__(self, max_seats=0):
-        self.max_seats = max_seats
-        self.plan = MockChargeBeePlan(max_seats)
+    def __init__(self, max_seats=0, max_api_calls=50000):
+        self.max_seats = (max_seats,)
```
Comment (matthewelwell) [inline api/organisations/tests/test_chargebee.py]: Why is this now a tuple?

Comment (matthewelwell) [reply api/organisations/tests/test_chargebee.py]: Looking at the rest of the code - I'm not sure these class attributes are even used are they?

Comment (gagantrivedi) [reply api/organisations/tests/test_chargebee.py]: It is used https://github.com/Flagsmith/flagsmith/pull/1007/files#diff-5e8e0e8015c37ed257e0cb30e55f43f67306a2d98c054d4601e03223aa4ba1f3L117, no?

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**